cfme/utils/template/base.py

```diff
@@ -254,6 +262,34 @@ def download_image(self):
         else:
             return True
 
+    @log_wrap('unzip image')
+    def unzip_image(self, archive_type='zip'):
```

Comment:
Consider making this part of download_image, called automatically when the downloaded file is `.zip` or `.tar.gz`. Then the `run` methods don't have to separately call this to unpack the archive.

Reply:
Done